extractor/src/test/resources/org/schabi/newpipe/extractor/services/youtube/extractor/stream/pewdiwpie/generated_mock_0.json

```diff
@@ -26,53 +124,141 @@
         "h3\u003d\":443\"; ma\u003d2592000,h3-29\u003d\":443\"; ma\u003d2592000,h3-T051\u003d\":443\"; ma\u003d2592000,h3-Q050\u003d\":443\"; ma\u003d2592000,h3-Q046\u003d\":443\"; ma\u003d2592000,h3-Q043\u003d\":443\"; ma\u003d2592000,quic\u003d\":443\"; ma\u003d2592000; v\u003d\"46,43\""
       ],
       "cache-control": [
-        "no-cache, no-store, max-age\u003d0, must-revalidate"
-      ],
-      "content-disposition": [
-        "attachment"
+        "private"
       ],
       "content-type": [
-        "application/json; charset\u003dutf-8"
+        "application/json; charset\u003dUTF-8"
       ],
       "date": [
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
there are unresolved conflicts in mocks

Reply:
I am aware of it, I didn't regenerate them.